### DISCUSSION TITLE
how to open editor for description when modifiyng task?

### DISCUSSION THREAD

**Discussion by TornaxO7** [GothenburgBitFactory/taskwarrior#3210] [Q&A] [answered]
Hello! Is it possible to add more text to the tasks by using an external editor? Because sometimes I'm changing a lot of the description and I don't want to rewrite a part of text again

**Comment by djmitche** [comment]
Yes, try `task edit`

**Reply by TornaxO7** [reply]
Is there no command to open the editor *only* for the description?

**Reply by djmitche** [reply, marked as answer]
No, sorry

**Reply by TornaxO7** [reply]
welp, better than nothing then. Thank you.
